static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -237,9 +237,7 @@ class IssueAlertOptions extends DeprecatedAsyncComponent<Props, State> {
       actions: [
         {
           ...ISSUE_ALERT_DEFAULT_ACTION,
-          ...(this.props.organization.features.includes('issue-alert-fallback-targeting')
-            ? {fallthroughType: 'ActiveMembers'}
-            : {}),
+          ...{fallthroughType: 'ActiveMembers'},
```

Comment:
i think in this case we can change ISSUE_ALERT_DEFAULT_ACTION to have `fallthroughType: 'ActiveMembers'` all the time